### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by Roundaround** [nutdotnet/Coco.Nut#9] [General]
At this point you announced you were starting this over a year ago with no commits or progress updates. Is this still being worked on?

**Comment by gbakeman** [comment]
Hi @Roundaround ,

I started up this project to start gathering ideas and plan for WinNUT's eventual replacement as the primary Windows app for monitoring a NUT server. I intended to only do some bug fixes for WinNUT before officially sunsetting it, although as I work on it more, I'm starting to get more involved and do larger projects on it. To be honest, I'm thinking I may try integrating the NUT.Net client library into WinNUT as a way to develop and polish the library before starting in earnest on a new client UI.

I apologize for the delay, but hope this will both end up leaving WinNUT in a polished state for people who want to keep using it, as well as prepare code libraries for use in the new iteration. Please consider leaving your feedback and ideas in this project, which will help me better plan it. In the meantime though, you can find continued support in the original WinNUT project, and eventually I hope to wrap it up and move onto this one.

**Reply by Roundaround** [reply]
No need to apologize! I just wanted to know what was going on. Thanks for filling me in!

**Comment by rederikus** [comment]
@gbakeman,  Thanks for the update.  I too was beginnig to wonder about this project.  No rush but thanks again.

**Comment by buzzwebly** [comment]
I'd like the ability to have it run as a service, so I can run it on my media server without having to be logged in.
